org.eclipse.xtext.ide.tests/testlang-src-gen/org/eclipse/xtext/ide/tests/testlanguage/validation/AbstractIndentationAwareUiTestLanguageValidator.java

```diff
@@ -20,5 +20,4 @@
 		result.add(org.eclipse.xtext.ide.tests.testlanguage.indentationAwareUiTestLanguage.IndentationAwareUiTestLanguagePackage.eINSTANCE);
 		return result;
 	}
-	
```

Comment:
Should we try to avoid that WS change instead?

Reply:
i think it was an optimization the the generator so we should reflect it here. otherwise we would have to sort it out forever.

Reply:
Agreed